ui-app/client/src/components/Gallery/galleryProperties.ts

```diff
@@ -0,0 +1,289 @@
+			{
+				name: 'Right',
+				displayName: 'right',
+				description: 'slide navigations button positioned on right',
+			},
+			{
+				name: 'Bottom',
+				displayName: 'bottom',
+				description: 'slide navigations button positioned on bottom',
+			},
+			{
+				name: 'Left',
+				displayName: 'left',
+				description: 'slide navigations button positioned on left',
+			},
+		],
+	},
+
+	{
+		name: 'autoPlay',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Auto Play',
+		description: 'Play automatically without interaction',
+		group: ComponentPropertyGroup.BASIC,
+		defaultValue: false,
+	},
+
+	{
+		name: 'slideSpeed',
+		schema: SCHEMA_NUM_COMP_PROP,
+		displayName: 'Gallery slide speed',
+		description: `Adjust the slide speed by giving the time in milliseconds.`,
+		group: ComponentPropertyGroup.ADVANCED,
+		defaultValue: 2000,
+	},
+
+	{
+		name: 'animationType',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Animation Type',
+		description: 'Animation Type',
+		editor: ComponentPropertyEditor.ENUM,
+		group: ComponentPropertyGroup.ADVANCED,
+		defaultValue: 'fadeover',
+		enumValues: [
+			{ name: 'slide', displayName: 'Slide', description: 'Slide' },
+			{ name: 'slideover', displayName: 'Slide Over', description: 'Slide Over' },
+			{ name: 'fadeover', displayName: 'Fade Over', description: 'Fade Over' },
+			{ name: 'fadeoutin', displayName: 'Fade Out In', description: 'Fade Out In' },
+			{ name: 'crossover', displayName: 'Cross Over', description: 'Cross Over' },
+		],
+	},
+
+	{
+		name: 'animationDuration',
+		schema: SCHEMA_NUM_COMP_PROP,
+		displayName: 'Animation Duration',
+		description: 'How long the animation should last.',
+		group: ComponentPropertyGroup.ADVANCED,
+		defaultValue: 2000,
+	},
+
+	{
+		name: 'easing',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Easing',
+		description: 'Easing',
+		editor: ComponentPropertyEditor.ENUM,
+		group: ComponentPropertyGroup.ADVANCED,
+		defaultValue: 'ease',
+		enumValues: [
+			{ name: 'linear', displayName: 'Linear', description: 'Linear' },
+			{ name: 'ease', displayName: 'Ease', description: 'Ease' },
+			{ name: 'ease-in', displayName: 'Ease In', description: 'Ease In' },
+			{ name: 'ease-out', displayName: 'Ease Out', description: 'Ease Out' },
+			{ name: 'ease-in-out', displayName: 'Ease In Out', description: 'Ease In Out' },
+			{ name: 'crossover', displayName: 'Cross Over', description: 'Cross Over' },
+		],
+	},
+
+	COMMON_COMPONENT_PROPERTIES.showInDesign,
+];
+
+const stylePropertiesDefinition: ComponentStylePropertyDefinition = {
+	'': [
+		COMPONENT_STYLE_GROUP_PROPERTIES.layout.type,
+		COMPONENT_STYLE_GROUP_PROPERTIES.position.type,
+		COMPONENT_STYLE_GROUP_PROPERTIES.spacing.type,
+		COMPONENT_STYLE_GROUP_PROPERTIES.typography.type,
+		COMPONENT_STYLE_GROUP_PROPERTIES.border.type,
+		COMPONENT_STYLE_GROUP_PROPERTIES.size.type,
+		COMPONENT_STYLE_GROUP_PROPERTIES.effects.type,
+		COMPONENT_STYLE_GROUP_PROPERTIES.background.type,
+	],
+};
```

Comment:
Add subcomponent styles for buttons, gallery images, thumbnail images, thumbnail containers, hover and other options

Reply:
added